plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -37,17 +38,30 @@ import {
   MongoInternalCommand,
   MongoInternalTopology,
   WireProtocolInternal,
+  CommandResult,
 } from './types';
 import { VERSION } from './version';
 
 const supportedVersions = ['>=3.3 <4'];
 
+const DEFAULT_CONFIG: MongoDBInstrumentationConfig = {
+  enhancedDatabaseReporting: false,
+};
+
 /** mongodb instrumentation plugin for OpenTelemetry */
 export class MongoDBInstrumentation extends InstrumentationBase<
   typeof mongodb
 > {
-  constructor(protected _config: MongoDBInstrumentationConfig = {}) {
-    super('@opentelemetry/instrumentation-mongodb', VERSION, _config);
+  constructor(config: MongoDBInstrumentationConfig = {}) {
+    super("@opentelemetry/instrumentation-mongodb", VERSION, Object.assign({}, DEFAULT_CONFIG, config));
```

Comment:
Did you the lint task? It should be using '' and not ""

Reply:
Had some issues with the linter, fixed this now :)